internal/rbd/controllerserver_test.go

```diff
@@ -67,3 +68,149 @@ func TestIsThickProvisionRequest(t *testing.T) {
 		t.Errorf("request should be for thick-provisioning: %s", req.Parameters["thickProvision"])
 	}
 }
+
+func TestIsMigrationVolHandle(t *testing.T) {
+	t.Parallel()
+	volID := "wrong_volume_handle"
+	req := &csi.DeleteVolumeRequest{
+		VolumeId: volID,
+	}
```

Comment:
this is not needed. use the volID defined at line 74?

Reply:
@Madhu-1 we are using `volID` defined at line 74 here, I am sorry, I didnt get the comment.

Reply:
why create req variable and send req.VolumeID to the isMigrationVolHandle?

```
volID := "wrong_volume_handle"
if isMigrationVolHandle(volID) {
	t.Error("request is not a migration delete request")
}
```